### DISCUSSION TITLE
【小调查】粉丝勋章数量

### DISCUSSION THREAD

**Discussion by andywang425** [andywang425/BLTH#546] [Polls]
这个投票纯图一乐，同时也会一定程度上决定BLTH以后是否要对粉丝勋章数量特别多的情况做优化。

如果你有多个B站账号，按拥有粉丝勋章数量最多的账号来填即可。

**Comment by MementoCute** [comment]
![image](https://github.com/user-attachments/assets/34f6daeb-b0f8-4c7c-883f-6577fa82857c)
5000

**Comment by 36EMiyuki** [comment]
![PixPin_2024-09-17_13-58-36](https://github.com/user-attachments/assets/b0ddcb8c-a862-4a8e-a8c6-8bc42d5a7034)
855个牌子

**Comment by huhanming0001** [comment]
![QQ截图20240917212259](https://github.com/user-attachments/assets/72800c2d-e8e8-4e89-8073-59956c5e1a87)
280

**Comment by jakaz1280** [comment]
![image](https://github.com/user-attachments/assets/a161c3a6-ca61-42f9-87a5-a4d7877579fe)
349
